src/etc/inc/unbound.inc

```diff
@@ -947,4 +953,38 @@ function unbound_local_zone_types() {
 	);
 }
 
+// Autoconfig EDNS buffer size
+function unbound_auto_ednsbufsize() {
+	global $g, $config;
```

Comment:
`$g` variable is not being used in this function

Reply:
fixed